kafka-connect/src/main/java/io/tabular/iceberg/connect/channel/Worker.java

```diff
@@ -93,6 +99,10 @@ private void save(SinkRecord record) {
         new TopicPartition(record.topic(), record.kafkaPartition()),
         new Offset(record.kafkaOffset() + 1, record.timestamp()));
 
+    sourceTxIds.put(
+        new TopicPartition(record.topic(), record.kafkaPartition()),
+        Utilities.extractTxIdFromRecordValue(record.value(), COL_TXID));
```

Comment:
this will overwrite the previous entry for the partition everytime (which is what we want i think). But do we want it to get set to -1 if we can't read the txid from the entry? It might make more sense to not write it if we can't extract it (so we don't overwrite the last one we had)